Mathlib/Tactic/Linter/DocString.lean

```diff
@@ -64,6 +73,9 @@ def docStringLinter : Linter where run := withSetOptionIn fun stx ↦ do
     -- `docString` contains e.g. trailing spaces before the `-/`, but does not contain
     -- any leading whitespace before the actual string starts.
     let docString ← try getDocStringText ⟨docStx⟩ catch _ => continue
+    if docString.trim.isEmpty then
+      Linter.logLintIf linter.style.docString.empty docStx m!"warning: this doc-string is empty"
+      continue
```

Comment:
```suggestion
```
is this needed?  I would have guessed that the default would be to `continue`.

Reply:
Oh, and actually trying this out reveals why the continue is useful: without it, an empty doc-string throws *three* errors:
```
warning: warning: this doc-string is empty

Note: This linter can be disabled with `set_option linter.style.docString.empty false`
---
warning: error: doc-strings should start with a single space or newline

Note: This linter can be disabled with `set_option linter.style.docString false`
---
warning: error: doc-strings should end with a single space or newline

Note: This linter can be disabled with `set_option linter.style.docString false`
```
while just one would suffice.